### ISSUE TITLE
Further improvements for the PhenoDigm parser

### ISSUE THREAD

**Issue by tskir** [opentargets/platform#1531]
* **General improvements**
  + [ ] A nice addition to the script would be to calculate the various quality control metrics. For example, which percentage of the original disease-model associations were eventually converted into the evidence strings, and what are the reasons for the failing records.
  + [ ] The static MGI and HGNC mapping files could be downloaded using Snakemake (once that PR is merged) instead of directly by the script.
  + [ ] Conduct more detailed investigations on the meaning of `disease_model_max_norm`, perhaps by reading the PhenoDigm paper or contacting the developers. Decide whether to filter on it or not.
* **Representation improvements**
  + [ ] For the 21.06 release, we must ensure that the platform backend & frontend are updated to reflect the updated evidence schema regarding mouse gene ID and name.
  + [ ] The platform website could/should provide more information on how the mouse model data are computed and what is the meaning of different fields. In particular, which phenotypes are presented.
  + [ ] It might be useful to get an input from someone who is working with mouse models a lot, to see how people are using this information; if they are satistifed; if they want additional fields, etc.
* **Ontology mappings**
  + [ ] As per discussions, `diseaseFromSourceMappedId` is currently **not** populated by the refactored pipeline. It is expected that this field is going to be populated by downstream pipelines. If this functionality is still not available in time for the 21.06 release, this functionality will have to be temporarily implemented (again) in the parser.

**Comment by tskir** [comment]
This is a follow-up issue from the major rewrite in https://github.com/opentargets/evidence_datasource_parsers/pull/80. Because we had to stop iterating at some point, the remaining points were collected here.

**Comment by d0choa** [comment]
We are not planning to work on this short term, but this ticket will be useful for future reference.
